manifests/o/OxygenCloud/odrive/1.0.7075/OxygenCloud.odrive.installer.yaml

```
@@ -0,0 +1,22 @@
+# Created with YamlCreate.ps1 v2.0.0 $debug=NVS1.7-1-5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: OxygenCloud.odrive
+PackageVersion: 1.0.7075
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: burn
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ProductCode: '{f73edcdf-4d4d-43b5-9758-6e8b9d1d5c2d}'
+Installers:
+- Architecture: x86
+  InstallerUrl: https://www.odrive.com/downloaddesktop?platform=win
```

Not sure if we should use this InstallerUrl instead?
```suggestion
  InstallerUrl: https://dba0rg1h0v09q.cloudfront.net/odrivesync.7075.exe
```